tests/api/route-services/samples.test.js

```diff
@@ -54,15 +83,46 @@ describe('tests for the samples service', () => {
 
     const getItemSpy = mockDynamoGetItem(jsData);
 
-    (new SamplesService()).getSampleIds('12345')
+    (new SamplesService()).getSampleIds('project-1')
       .then((data) => {
         expect(data).toEqual(jsData);
         expect(getItemSpy).toHaveBeenCalledWith({
           TableName: 'samples-test',
-          Key: { experimentId: { S: '12345' } },
+          Key: { experimentId: { S: 'project-1' } },
           ProjectionExpression: 'samples.ids',
         });
       })
       .then(() => done());
   });
+
+  it('udpateSamples work', async (done) => {
```

Comment:
typo in `udpateSamples ` too

Reply:
apart from unit test for whether the update works, we should test for the behaviour for any corner cases that are possible, for example: What if we try to update non-existent sample? What if we pass empty samples entry in the update: is it going to delete all existent samples? What if dynamoDB was down and the actual update didn't happen?